bundles/af-core/pom.xml

```diff
@@ -457,8 +457,11 @@
         <!-- AEM and forms sdk versions -->
         <!-- the below order of dependencies is very critical -->
         <dependency>
-            <groupId>com.adobe.aem</groupId>
+            <groupId>com.adobe.aemfd</groupId>
             <artifactId>aem-forms-sdk-api</artifactId>
+            <version>2024.04.01.01-SNAPSHOT</version>
```

Comment:
No snapshot version, please merge your changes and create a release by talking to subodh

Reply:
I had a conversation with @nit23uec regarding this.
The SDK isn't released until the release ceremony happens. Will we explicitly release the forms-sdk for this customer and use the relevant version?

For now, I have embedded the SDK in the PR itself, which can be kept as a separate branch to deliver code to the customer. Do we need to change the approach here ?

Reply:
In past, we have made a release in the current state, once cq-guides are merged. Please talk to Subodh on how to create an intermittent release

Reply:
we should release an sdk on maven central only if its required by customer code. if its only required by core components, then we should prefer the inlining route.